runtime/compiler/runtime/Runtime.cpp

```diff
@@ -167,6 +170,29 @@ void* TR_RuntimeHelperTable::getFunctionPointer(TR_RuntimeHelper h)
 
 void* TR_RuntimeHelperTable::getFunctionEntryPointOrConst(TR_RuntimeHelper h)
    {
+#if defined(J9VM_OPT_JITSERVER) && defined(TR_HOST_POWER)
```

Comment:
People reading the code may wonder why we do this only for Power, so it's better to have a comment explaining why